opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -228,7 +226,7 @@ class Tracer(abc.ABC):
     def start_span(
         self,
         name: str,
-        parent: ParentSpan = CURRENT_SPAN,
+        parent: typing.Optional[Context] = None,
```

Comment:
I'm wondering if the variable is worth renaming. "parent" was vague to begin with, and if you look at any descriptions of how tracing works, one would immediately think "parent" refers to the span (since you specifically call the span a child of the parent span).

This would also be useful to rename since new API users will get exceptions when their existing code passes an argument as parent. Rather than get a new slightly confusing error ("why is the API now rejecting this previously valid value?"), they will get an argument not supported error, and look up the docs to see the variable changed.

`parent_context` is the first thing that comes to mind, but I worry a user would get it confused with the parent SpanContext.

Reply:
@toumorokoshi yeah I had a similar thought around the name. Looking at some of the other implementations, it appears `parent` and `parent_context` are the current names used. I'm happy to change it to parent_context as it's at least a bit more meaningful than parent. The alternative i can think of would be to just call it `context`

Reply:
Either sounds great!

Reply:
Thanks for the feedback, went with `context`.